daemon/lib/source/DobbyManager.cpp

```diff
@@ -2221,7 +2221,6 @@ bool DobbyManager::onPreDestructionHook(const ContainerId &id,
 void DobbyManager::handleContainerTerminate(const ContainerId &id, const std::unique_ptr<DobbyContainer>& container, const int status)
 {
     AI_LOG_FN_ENTRY();
-
```

Comment:
If you didn't change anything around this place don't make style adjustments as it will result in possible merge conflicts.